gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_4_0/01_store_federated_apis.yml

```diff
@@ -12,3 +12,4 @@ databaseChangeLog:
         - dropNotNullConstraint:
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.